docs/en/administration/sync.md

```diff
@@ -1,4 +1,243 @@
+
+Assume that we have these following storages.
+
+1. **Object Storage A** <span id="bucketA" />
+   - Bucket name: aaa
+   - Endpoint: `https://aaa.s3.us-west-1.amazonaws.com`
+
+2. **Object Storage B** <span id="bucketB" />
+   - Bucket name: bbb
+   - Endpoint: `https://bbb.oss-cn-hangzhou.aliyuncs.com`
+   
+3. **JuiceFS File System** <span id="bucketC" />
+   - Metadata Storage: `redis://10.10.0.8:6379/1`
+   - Object Storage: `https://ccc-125000.cos.ap-beijing.myqcloud.com`
+   
+All of the storages share the same **secret key**:
+
+- **ACCESS_KEY**: `ABCDEFG`
+- **SECRET_KEY**: `HIJKLMN`
+
+### Synchronize between Object Storage and JuiceFS
+
+The following command synchronizes `movie` directory on [Object Storage A](#bucketA) to [JuiceFS File System](#bucketC).
+
+```shell
+# mount JuiceFS
+sudo juicefs mount -d redis://10.10.0.8:6379/1 /mnt/jfs
+# synchronization
+juicefs sync s3://ABCDEFG:HIJKLMN@aaa.s3.us-west-1.amazonaws.com/movies/ /mnt/jfs/movies/
+```
+
+The following command synchronizes `images` directory from [JuiceFS File System](#bucketC) to [Object Storage A](#bucketA).
+
+```shell
+# mount JuiceFS
+sudo juicefs mount -d redis://10.10.0.8:6379/1 /mnt/jfs
+# synchronization
+juicefs sync /mnt/jfs/images/ s3://ABCDEFG:HIJKLMN@aaa.s3.us-west-1.amazonaws.com/images/
+```
+
+### Synchronize between Object Storages
+
+The following command synchronizes all of the data on [Object Storage A](#bucketA) to [Object Storage B](#bucketB).
+
+```shell
+juicefs sync s3://ABCDEFG:HIJKLMN@aaa.s3.us-west-1.amazonaws.com oss://ABCDEFG:HIJKLMN@bbb.oss-cn-hangzhou.aliyuncs.com
+```
+
+## Advanced Usage
+
+### Incremental and Full Synchronization
+
+The subcommand `sync` works incrementally by default. That is, compares the differences between the source and target paths, and then synchronizes only the differences. You can add option `--update` or `-u` to keep updated the `mtime` of the synchronized directories and files.
+
+To do a full synchronization, i.e. always synchronize no matter whether the destination files exist or not, you can add option `--force-update` or `-f`. For example, the following command fully synchronizes `movies` directory from [Object Storage A](#bucketA) to [JuiceFS File System](#bucketC).
+
+```shell
+# mount JuiceFS
+sudo juicefs mount -d redis://10.10.0.8:6379/1 /mnt/jfs
+# full synchronization
+juicefs sync --force-update s3://ABCDEFG:HIJKLMN@aaa.s3.us-west-1.amazonaws.com/movies/ /mnt/jfs/movies/
+```
+
+### Pattern Matching
+
+The pattern matching function of the `sync` subcommand is similar to that of `rsync`, which allows you to exclude or include a certain classes of files by rules, and synchronize any set of files by combining multiple rules. Now we have these following rules available.
+
+- Patterns ending with `/` will only match directories, otherwise, they will match files, links or devices;
+- Patterns containing `*`, `?` or `[` will match as wildcards, otherwise, they will match as regular strings;
```

Comment:
```suggestion
- Patterns containing `*`, `?` or `[` follows wildcard pattern to match; otherwise, matching uses regular strings.
```
这句好难翻译，尝试了修改了一下，不知道理解对不对 :P

Reply:
```suggestion
- Patterns containing `*`, `?` or `[` match as wildcards, otherwise, they match as regular strings;
```